blackbox/docs/src/crate/tests.py

```diff
@@ -118,6 +118,7 @@ def java_transform(s):
 class ConnectingCrateLayer(CrateLayer):
 
     def __init__(self, *args, **kwargs):
+        kwargs['settings'] = {};
```

Comment:
Nope:
https://github.com/crate/crate/pull/4801/files#diff-0fa31503051210744ee04c174af5ddb6L160

Should I add a check: if kwargs is None: ?